bitnami/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.23.0
+version: 6.24.0
```

Comment:
After changing all those settings in the values.yml I will go for a mayor version update.

Reply:
I have doubts about it. The upgrade process seems to go smoothly. Prometheus detects the new endpoint on the fly.